### ISSUE TITLE
Enhancement: add an attribute to report mote transmit power/EIRP

### ISSUE THREAD

**Issue by danak6jq** [helium/mappers#122]
I built a clone of an LGT-92 using a Sparkfun SparkX ProRF 1W radio, with an eByte E19-915M30S, went mobile with a mag-mount on the car roof. It occurs to me this has a much higher EIRP than 'normal' trackers and will influence the mapping data.

Perhaps an optional attribute to report EIRP makes sense, so collected data might be optionally displayed or otherwise indicated. Same thing could be said for antenna selection.

I know there's a whole slippery slope about how *do* I estimate EIRP accurately, I don't mean to open that can of worms.

In the meantime, I'm using a real LGT-92 with the internal antenna for hex-mapping.

**Comment by danak6jq** [comment]
TL;DR: motes operating on an 8-channel US sub-band are limited to +20dBm regardless of mote capability.
Even shorter TL;DR: nothing to see here, move on

US regulations allow up to +30dBm when using 50 or more channels, and ~ +21dBm when using fewer than 50 (but more than 4) channels. The Semtech/Stackforce reference port indeed limits TxPower to 20dBm when using fewer than 50 channels. So even a "high power" tracker should be operating at +20dBm when using an 8-channel US sub-band.

Perhaps antenna class (internal, short stub, vehicle exterior) could be reported?
